### PR TITLE
Clarify the relationship of ASP.NET Identity, IdentityServer4 and Microsoft identity platform

### PR DESCRIPTION
**Pull request by PeterDraex**
I found it challenging to understand the relationship of ASP.NET Identity, Duende IdentityServer4 and Microsoft identity platform. I hope this change can make it easier for others.

### CODE REVIEW

**Comment by PeterDraex** [comment]
@Rick-Anderson I think you'd be the right reviewer here :)

File: aspnetcore/security/authentication/identity.md [hunk 1]
```diff
@@ -32,7 +33,7 @@ In this topic, you learn how to use Identity to register, log in, and log out a
 [Microsoft identity platform](/azure/active-directory/develop/) is:
 
 * An evolution of the Azure Active Directory (Azure AD) developer platform.
-* Unrelated to ASP.NET Core Identity.
+* An alternative identity solution for authentication and authorization in ASP.NET Core applications.
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/identity.md:36]:
```suggestion
* An alternative identity solution for authentication and authorization in ASP.NET Core apps.
* Not related to ASP.NET Core Identity.
```

File: aspnetcore/security/authentication/identity.md [hunk 2]
```diff
@@ -277,9 +279,8 @@ Identity is typically configured using a SQL Server database to store user names
 In this topic, you learn how to use Identity to register, log in, and log out a user. Note: the templates treat username and email as the same for users. For more detailed instructions about creating apps that use Identity, see [Next Steps](#next).
 
 [Microsoft identity platform](/azure/active-directory/develop/) is:
-
 * An evolution of the Azure Active Directory (Azure AD) developer platform.
-* Unrelated to ASP.NET Core Identity.
+* An alternative identity solution for authentication and authorization in ASP.NET Core applications.
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/identity.md:283]:
```suggestion
* An alternative identity solution for authentication and authorization in ASP.NET Core apps.
* Not related to ASP.NET Core Identity.
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
Thanks for the contribution. Be sure to check out your name as a contributor about the 3rd of next month on the [What's new page](https://docs.microsoft.com/en-us/aspnet/core/whats-new/) in the **Community contributors** section at the bottom.

**Pull request merged**